src/Core/Domain/Entity/TelemetryEvent/TelemetryEventMetadataInterface.php

```diff
@@ -17,10 +17,10 @@ public function getDeveloperEmail(): ?string;
     /**
      * @return string|null
      */
-    public function getDeveloperGithubAccount(): ?string;
+    public function getProjectName(): ?string;
```

Comment:
Why can it be null?

Reply:
In case we unable to fetch the value. All the telemetry data that is come from the not controlled project should be nullable